scripts/upgrade

```diff
@@ -132,13 +55,6 @@ ynh_script_progression --message="Starting a systemd service..."
 
 ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line_match="Proxy listening"
 
-#=================================================
```

Comment:
Here, ynh_systemd_action start fails because it’s already started. You should stop it at the beginning of the upgrade.